client/components/Events/EventEditor/index.jsx

```diff
@@ -60,7 +60,7 @@ export class EventEditorComponent extends React.Component {
     }
 
     componentDidMount() {
-        if (!get(this.props, 'navigation.scrollToViewItem')) {
+        if (!get(this.props, 'navigation.scrollToViewItem') && this.dom.slugline) {
```

Comment:
If the form does not have a **slugline** input field, then we should focus the **name** input field.